website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -89,42 +97,55 @@ in `role_binding.yaml` are used to grant the operator permissions to access and
 For the purposes of this doc, the other RBAC manifests `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`,
 and `auth_proxy_*.yaml` are not relevant to changing the operator's resource permissions.
 
-### Changing the permissions
+### Changing the permissions to Namespaced
 
 To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use `Role`s
-and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
-
-[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/memcached_controller.go`)
-are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default
- markers don't specify a `namespace` property and will result in a `ClusterRole`.
+=======
 
-Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role`
- instead of a `ClusterRole`.
+- Inform the Namespace to the [Manager][ctrl-manager] 
 
-Replace:
+By default, the [Manager][ctrl-manager] does not have any namespace specified in `main.go`, and hence it will watch all the namespaces. In order to restrict the controllers to watch a specific namespace, specify it while creating the manager. Update the `NewManager` to inform the Namespace, in our `Memcahced` example it would like: 
 
 ```go
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+    Namespace:          "memcached-operator-system", // operator namespace
+})
+...
```

Comment:
The doc is about operator-scope. The RBAC changes the permissions of the operator,  however, if a user is changing the scope then would be recommend also add the ns to the manager since it will make the manager ONLY look for and cache just resources from the specified ns instead of all cluster which is the default behaviour. 

So, it is one of the required/recommend steps when a user is changing the scope from cluster to namespaced and then, because of this it is here. PS.: Note that the change here is only to update the example code which is not accurate at all.